pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -70,6 +194,15 @@ export function make_btrfs_subvolume_page(parent, volume, subvol) {
         });
     }
 
+    // refactor to always be mount_point_in_parent?
```

Comment:
// refactor to always be mount_point_in_parent?

Yes, imo.

Reply:
I noticed that if it's just a /foo it where / is the rood id (not mounted) it always returned `/foo` so it already seems to do what the `mounted` condition is for.

Reply:
Re-reading this and testing, it is actually not true. mount_point_in_parent does what it says on the tin and I would like it to stay the same honestly. For the delete feature we likely also only need the mount_point_in_parent for example if I want to delete `bar` which is in `/foo/bar/bob`.

Then again maybe this is always the same as you need the path under the current subvolume always when deleting...

Reply:
Ok, delete needs it to be explicitly the mount point for the parent and not it's own mount point. So I say we don't do this.

Reply:
Agreed.